AppleAlarm/Alarm.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Alarm.swift
+//  AppleAlarm
+//
+//  Created by Alexander McLean on 1/30/16.
+//  Copyright © 2016 Antar Vasi. All rights reserved.
+//
+
+import Foundation
+
+class Alarm : NSObject {
+    var name: String
```

Comment:
Where are all the extra member variables you said you added to this class?  I don't see them listed here.